proto/ddl_service.proto

```diff
@@ -444,6 +445,21 @@ message CommentOnResponse {
   uint64 version = 2;
 }
 
+message TableSchemaChange {
+  enum TableChangeType {
+    UNSPECIFIED = 0;
+    ALTER = 1;
```

Comment:
Will we support `ALTER ... RENAME` from upstream?

Reply:
According to the doc, the variants must be either `ALTER`, `CREATE`, or `DROP`. Since this message is named `TableSchemaChange`, I suppose the `change_type` is implied and we don't need field.

On the other hand, this means that all kinds of table altering will be categorized into `ALTER`. At the same time, we get a **complete** schema in `columns`. So it might be our responsibility to detect what the actual change is.